deploy/charts/approver-policy/README.md

```diff
@@ -255,6 +255,22 @@ For more information, see [AWS EKS](https://cert-manager.io/docs/installation/co
 > ```
 
 This value may need to be changed if `hostNetwork: true`
+#### **priorityClassName** ~ `string`
+
+Configure the priority class of the Pod.  
+  
+For more information, see:  
+* [Guaranteed Scheduling For Critical Add-On Pods](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/)  
+* [Protect Your Mission-Critical Pods From Eviction With PriorityClass](https://kubernetes.io/blog/2023/01/12/protect-mission-critical-pods-priorityclass/)  
```

Comment:
Other charts link to [PriorityClass](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass), but I think these links are more useful.

https://github.com/cert-manager/trust-manager/blob/d7fe0bb006a771252ab070506187b70759c6bbc0/deploy/charts/trust-manager/values.yaml#L130-L131